src/renderer/hooks/useSubagentMessages.ts

```
@@ -0,0 +1,54 @@
+  // Subscribe narrowly so unrelated cache mutations don't re-render us.
+  const messages = useStore((s) => s.subagentMessageCache.get(subagentId)?.messages ?? null);
+  const isLoading = useStore((s) => s.loadingSubagentIds.has(subagentId));
+  const error = useStore((s) => s.subagentMessageErrors.get(subagentId) ?? null);
+  const loadSubagentMessages = useStore((s) => s.loadSubagentMessages);
+
+  useEffect(() => {
+    if (!enabled) return;
+    if (!projectId || !sessionId) return;
+    if (messages !== null) return; // already cached
+    if (isLoading) return; // already in flight
+    void loadSubagentMessages(projectId, sessionId, subagentId);
```

ESLint

[error] Remove this use of the "void" operator.

(sonarjs/void-use)